cosmos/launchpad/construction_derive.go

```diff
@@ -4,15 +4,20 @@ import (
 	"context"
 
 	"github.com/coinbase/rosetta-sdk-go/types"
-	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
 func (l Launchpad) ConstructionDerive(ctx context.Context, r *types.ConstructionDeriveRequest) (*types.ConstructionDeriveResponse, *types.Error) {
 	if r.PublicKey.CurveType != "secp256k1" {
 		return nil, ErrUnsupportedCurve
 	}
 
+	config := sdk.GetConfig()
```

Comment:
I am not sure, but I don't think this is the correct point to do it. Given it is a singleton I think the SetBech32 prefixx should be done when the application is called. Maybe on main.

Reply:
yeah moved it to launchpad constructor